### ISSUE TITLE
Discussion template needs extra heading below Glossary

### ISSUE THREAD

**Issue by jaredmorgs** [thegooddocsproject/templates#42]
Gloria Ma
12:44 14 Nov
Should have another top-level header here? Otherwise, this just looks like part of the Glossary.

**Suggestion**

About <subject>
For example, "About kubernetes containers"

**Comment by shreyas1599** [comment]
@jaredmorgs Do you mean like dividing the glossary into subsections? So the result would look something like this?

> About Topic1
_item1
item2_

> About Topic2
_item1
item2_

**Comment by jaredmorgs** [comment]
I also noticed this template has reference to Confluence in the comment at the top of the raw document source. Should remove that as well in the process of doing this ticket.

Sorry @shreyas1599 there isn't really a lot of context around this bug (my fault, because I created it from the google feedback docs). 

Maybe roll this fix about the comment into some other work you (or someone else taking this ticket) does for the Discussions template.

https://raw.githubusercontent.com/thegooddocsproject/templates/master/discussion/template-discussion.adoc
